CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v1.7.2 / 2019-08-05
+
+* [CHANGE] Revert "add kube_*_annotations metrics for all objects" #859
```

Comment:
Is it me or does this text look inconsistent?

Reply:
Fixed, PTAL thanks!